### PR TITLE
Feature/setup

### PR DESCRIPTION
**Pull request by sazukeR**
Here I present the first configurations related to the portfolio project 🔥 _ 🔥 

linter-html-running 🦾

linter-css-running 👍

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-18.04
```
Comment (hendridg) [inline .github/workflows/linters.yml]: Hi @sazukeR , well done!

Please change the `ubuntu-version` from `ubuntu-18.04` to `ubuntu-22.04`

File: .github/workflows/linters.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+          node-version: "12.x"
+      - name: Setup Webhint
+        run: |
+          npm install hint --save-dev
+          [ -f .hintrc ]
+      - name: Webhint Report
+        run: npx hint .
+  stylelint:
+    name: Stylelint
+    runs-on: ubuntu-18.04
```
Comment (hendridg) [inline .github/workflows/linters.yml]: Please change the `ubuntu-version` from `ubuntu-18.04` to `ubuntu-22.04`

File: .github/workflows/linters.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
+    name: Webhint
+    runs-on: ubuntu-18.04
```
Comment (hendridg) [inline .github/workflows/linters.yml]: Please change the `ubuntu-version` from `ubuntu-18.04` to `ubuntu-22.04`

**Review by hendridg — CHANGES_REQUESTED** [review]
# :recycle: Changes Required :recycle: 

Great job, please make the changes I suggest below

File: .github/workflows/linters.yml (hunk 3 above)
Comment (sazukeR) [reply .github/workflows/linters.yml]: i changed the file that you specified me 👍 (ln 11, 23, 37)
changes in yml file

the linters are runnings well
hint
stylelint

**Pull request merged**